src/app/errors/components/ErrorBoundary.tsx

```diff
@@ -56,6 +60,18 @@ class ErrorBoundary extends React.Component<Props, State> {
     this.props.recordError(error);
   }
 
+  public componentDidMount() {
+    if (windowDefined()) {
+      window?.addEventListener('unhandledrejection', this.handleRejection);
```

Comment:
if you just do the `typeof` check here in the if statement, you won't need the `?`, i don't love the creation of the extra code path